### ISSUE TITLE
split modules out into own repos.

### ISSUE THREAD

**Issue by swarren83** [rackspace-infrastructure-automation/mgcp-terraform-modules#4]
Each module code should be maintained in it's own repo in order to adhere to versioning standards.  Each time a PR is merged the version of the module should be bumped with a git tag.

**Comment by niven01** [comment]
Agreed. There is a discussion in the UK between FAWS, Azure and GCP build engineers for a location for this coming up

Should we just look at replicating FAWS and use repos in the `rackspace-infrastructure-automation` org?

Using the naming scheme `gcp-terraform-<module-name>`

**Comment by swarren83** [comment]
Yes, I have the ability to create repositories and will do so once we agree on the naming scheme.  I second `gcp-terraform-<module-name>`.

@p-at-rax and @upodroid Please weigh in on this.
